Plan/common/src/main/resources/assets/plan/locale/locale_JA.txt

```diff
@@ -135,30 +135,30 @@ HTML - LABEL_LONE_JOINS                         || 一人での接続
 HTML - LABEL_LONE_NEW_JOINS                     || 新しく一人での参加
 HTML - LABEL_LONGEST_SESSION                    || 最長接続時間
 HTML - LABEL_LOW_TPS                            || TPSの低下値
-HTML - LABEL_MAX_FREE_DISK                      || Max Free Disk
-HTML - LABEL_MIN_FREE_DISK                      || Min Free Disk
+HTML - LABEL_MAX_FREE_DISK                      || ディスクの最大空き容量
+HTML - LABEL_MIN_FREE_DISK                      || ディスクの最低空き容量
 HTML - LABEL_MOB_DEATHS                         || Mobによって殺された回数
 HTML - LABEL_MOB_KDR                            || Mobに対してのKDR
 HTML - LABEL_MOB_KILLS                          || Mobを殺した回数
 HTML - LABEL_MOST_ACTIVE_GAMEMODE               || 最も有効になっているゲームモード
 HTML - LABEL_NAME                               || 名前
-HTML - LABEL_NEW                                || 新しい
+HTML - LABEL_NEW                                || New
```

Comment:
Is this an intentional change?

Reply:
Yes, in Japan we use simple English to get attention or make things easier.